samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/shoppingcarts/config/CommandsConfig.java

```diff
@@ -0,0 +1,98 @@
+package io.eventdriven.ecommerce.shoppingcarts.config;
+
+import com.eventstore.dbclient.EventStoreDBClient;
+import io.eventdriven.ecommerce.core.commands.CommandHandler;
+import io.eventdriven.ecommerce.core.entities.EntityStore;
+import io.eventdriven.ecommerce.pricing.ProductPriceCalculator;
+import io.eventdriven.ecommerce.pricing.RandomProductPriceCalculator;
+import io.eventdriven.ecommerce.shoppingcarts.Events;
+import io.eventdriven.ecommerce.shoppingcarts.ShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.addingproductitem.AddProductItemToShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.canceling.CancelShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.confirming.ConfirmShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.opening.OpenShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.removingproductitem.RemoveProductItemFromShoppingCart;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.context.annotation.ApplicationScope;
+import org.springframework.web.context.annotation.RequestScope;
+
+import java.util.Optional;
+
+@Configuration
+public class CommandsConfig {
+  @Bean
+  @RequestScope
```

Comment:
I don't understand why do you use `@RequestScope` in all places where is used in the codebase. For me, CommandHandlers / QueryHandlers etc. should be stateless, so singleton is perfectly fine for that :)

Reply:
These classes are now removed. I merged spreaded beans config for commands and queries into the Shopping cart application service (ShoppingCartService) in https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/dbc9ca7fc2fea6f78bebfc107eb28fc88fac7ed1.